runtimes/pythonrt/local_runner.go

```diff
@@ -145,6 +144,32 @@ func (r *LocalPython) Start(pyExe string, tarData []byte, env map[string]string,
 }
 
 func (r *LocalPython) Health() error {
+	if _, err := os.FindProcess(r.proc.Pid); err != nil {
```

Comment:
IMO do this after `r.proc.Signal(syscall.Signal(0))` and only if there's an error.

Reply:
done